src/whisper_at_train/run_combined_training.sh

```
@@ -7,49 +11,83 @@
 
-lr=1e-6
-freqm=0
-timem=10
-mixup=0.5
-batch_size=48
-model=whisper-high-lw_tr_1_8 #whisper-high-lw_tr_1_8 (tl-tr, lr=5e-5) whisper-high-lw_down_tr_512_1_8 (tl-tr-512, w/ low-dim proj, lr=1e-4)
-model_size=large-v1
+# Activate poetry environment (modify path as needed)
+VENV_PATH="/home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate"
+if [[ -f "$VENV_PATH" ]]; then
+    source "$VENV_PATH"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)